packages/httpsnippet-client-api/src/index.ts

```diff
@@ -95,7 +91,7 @@ const client: Client<APIOptions> = {
   convert: ({ cookiesObj, headersObj, postData, queryObj, url, ...source }, options) => {
     const opts = {
       ...options,
-    };
+    } as APIOptions;
```

Comment:
Eventhough all of our `APIOptions` are required for this snippet `httpsnippet` core marks **all** client options as optional so doing `new Oas(opts.apiDefinition)` thought we were trying to pass `new Oas(undefined)`. Forcefully re-casing options as options here fixes that.